src/Form/Base/BaseInputChoice.php

```
@@ -0,0 +1,199 @@
+     * @param string|Stringable $content Content to be rendered inside the tag.
+     * Note: The `<input>` element is a void element and does not render inner content. Use attributes such as `value`
+     * to configure the input's value.
+     * @param array $tokenValues Additional token values for template rendering.
+     *
+     * @return string Rendered HTML for the inline element.
+     *
+     * @phpstan-param mixed[] $tokenValues
+     *
+     * @phpstan-return string
+     */
+    private function buildElement(string|null $id, string|Stringable $content = '', array $tokenValues = []): string
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)